binding.cc

```diff
@@ -425,7 +444,6 @@ NAPI_METHOD(utp_napi_close) {
   err = uv_udp_recv_stop(&(self->handle));
   if (err < 0) UTP_NAPI_THROW(err)
 
-  uv_close((uv_handle_t *) &(self->handle), on_uv_close);
```

Comment:
This handle is now closed serially. If I leave his in parallel, it causes a segfault. Probably a timing error, but I couldn't figure out its cause.